deploy/examples/cluster.yaml

```
@@ -70,6 +70,8 @@ spec:
     # port: 8443
     # serve the dashboard using SSL
     ssl: true
+    # prometheusEndpoint: <protocol>://<prometheus-host>:<port>
```

Please add a one-line comment for each setting, probably the same comment that is in the bullet list I added above
```suggestion
    # <comment>
    # prometheusEndpoint: <protocol>://<prometheus-host>:<port>
```